### ISSUE TITLE
add get_statevector support for BraketLocalBackend

### ISSUE THREAD

**Issue by JordanAWS** [qiskit-community/qiskit-braket-provider#47]
Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/qiskit-community/qiskit-braket-provider?pullRequest=48) before we can accept your contribution.

Currently `BraketLocalBackend().run(circuit).result().get_statevector()` will fail. The state vector result type is available for the Braket Local Simulator: https://github.com/aws/amazon-braket-sdk-python/blob/main/examples/bell_result_types.py#L40

**Comment by manulpatel** [comment]
Hello @JordanAWS! I new to qiskit-bracket and would like to work on this issue. Could you please assign it to me and provide if any further hints?

**Comment by JordanAWS** [comment]
Thanks @manulpatel! Just assigned! Let me know if you have any specific questions or if the prompt is unclear :)

**Comment by manulpatel** [comment]
Thanks @JordanAWS for assigning me! But it seems @IceKhan13 has already done the work. You may consider him.

**Comment by JordanAWS** [comment]
Ah, good point. We will review his PR.

**Comment by dakk** [comment]
> Ah, good point. We will review his PR. 
> 

Thank you Jordan

**Comment by dakk** [comment]
Since the PR has been merged, should be the issue closed @JordanAWS ?
